post/release-0-2.qmd

```diff
@@ -14,39 +21,123 @@ since the last specification release.
 
 The GeoArrow specification establishes conventions for representing spatial data
 in [Apache Arrow](https://arrow.apache.org) formats (e.g., C Data Interface,
-Serialized IPC) and implementations (e.g., PyArrow, Arrow C++, arrow-rs).
+Serialized IPC) and implementations (e.g., [PyArrow][pyarrow], [Arrow C++][arrow-cpp], [arrow-rs]).
 It is separated into two parts:
 
-- A set of [extension type](https://geoarrow.org/extension-types.html) definitions
-  to propagate type-level spatial metadata (e.g., coordinate reference system, edge interpolation)
-- A set of [memory layouts](https://geoarrow.org/format.html) to represent geometry
+[pyarrow]: https://arrow.apache.org/docs/python/index.html
+[arrow-cpp]: https://arrow.apache.org/docs/cpp/index.html
+[arrow-rs]: https://docs.rs/arrow/latest/arrow/
+
+- A set of [memory layouts](../format.html) to represent geometry
   arrays using features of the Arrow type system/format (e.g., points can be
   represented as a `struct<x: double, y: double>`) instead of serialized into
-  well-known binary or some other binary format.
+  [well-known binary][wkb-spec] or some other binary format.
+- A set of [extension type](../extension-types.html) definitions
+  to propagate type-level spatial metadata (e.g., coordinate reference system, edge interpolation)
+
+[wkb-spec]: https://libgeos.org/specifications/wkb/#iso-wkb
 
-In addition to some minor updates to improve clarity, GeoArrow 0.2 includes four
+## Notable changes
+
+In addition to some minor updates to improve clarity, GeoArrow 0.2 includes a few
 notable changes.
 
-First, the language defining the content of the `"crs"` key in the extension metadata
+### New array types
+
+The GeoArrow 0.2 specification formalizes three new array types:
+
+- The ["box" memory layout](../format.html#box) represents an array of axis-aligned bounding boxes, represented as an Arrow struct.
+- The ["geometry" memory layout](../format.html#geometry) allows arbitrary geometries to be
+  represented within a single array, even when the geometry type differs across rows. It's implemented as an [Arrow union][arrow-union-layout] of the other native encodings.
```

Comment:
For some reason the hard wrapping of my editor wasn't working well with the quarto extension enabled. Feel free to edit my hard wrapping to your liking